### PR TITLE
add dark mode color sets

### PR DESCRIPTION
**Pull request by hsurf22**
Screenshot 2021-12-07 at 15-06-13 Storybook

### CODE REVIEW

File: src/assets/styles/vuetifyTheme.js [hunk 1]
```diff
@@ -103,33 +103,33 @@ export default {
...
-        orangeMedium: '#FBDBA7',
-        orangeLight: '#FEF4E5',
-        disabledPrimary: '#CECECE',
-        disabledMedium: '#E0E0E0',
-        disabledLight: '#F6F6F6',
+        whiteBackground: '#192439',
+        whitePopup: '#202b43',
+        textDark: '#ccd4e6',
+        textMedium: '#626e86',
+        textLight: '#3a4562',
```
Comment (olgakup) [inline src/assets/styles/vuetifyTheme.js:114]: Hey text light is 
 3a4561

File: src/assets/styles/vuetifyTheme.js [hunk 2]
```diff
@@ -103,33 +103,33 @@ export default {
...
+        greyPrimary:'#5a678a',
+        greyMedium: '#323d57',
+        greyLight: '#26324a',
```
Comment (olgakup) [inline src/assets/styles/vuetifyTheme.js:117]: this one is #263149

File: src/assets/styles/vuetifyTheme.js [hunk 3]
```diff
@@ -103,33 +103,33 @@ export default {
...
+        greenPrimary: '#06d9b9',
+        greenMedium: '#107e7a',
```
Comment (olgakup) [inline src/assets/styles/vuetifyTheme.js:119]: greenMedium is  #06D9B9

File: src/assets/styles/vuetifyTheme.js [hunk 4]
```diff
@@ -103,33 +103,33 @@ export default {
...
+        greenLight: '#154853',
+        bluePrimary: '#6292eb',
+        blueMedium: '#355288',
+        blueLight: '#283a5d',
+        redPrimary: '#ff5d70',
```
Comment (olgakup) [inline src/assets/styles/vuetifyTheme.js:124]: redPrimary is 
#FF5E70

File: src/assets/styles/vuetifyTheme.js [hunk 5]
```diff
@@ -103,33 +103,33 @@ export default {
...
+        redMedium: '#8c4155',
```
Comment (olgakup) [inline src/assets/styles/vuetifyTheme.js:125]: redMedium is #FF5E70

**Review by olgakup — CHANGES_REQUESTED** [review]

File: src/assets/styles/vuetifyTheme.js (hunk 1 above)
Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Fixed

File: src/assets/styles/vuetifyTheme.js (hunk 2 above)
Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Fixed

File: src/assets/styles/vuetifyTheme.js (hunk 3 above)
Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Fixed

File: src/assets/styles/vuetifyTheme.js (hunk 4 above)
Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Fixed

File: src/assets/styles/vuetifyTheme.js (hunk 5 above)
Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Fixed

Comment (olgakup) [reply src/assets/styles/vuetifyTheme.js]: I think you changed redPrimary and not red medium

File: src/assets/styles/vuetifyTheme.js (hunk 3 above)
Comment (olgakup) [reply src/assets/styles/vuetifyTheme.js]: I think you changed _greenPrimary_ instead if **greenMedium**

Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Screenshot from 2021-12-21 16-11-36
Screenshot from 2021-12-21 16-11-41

Green Primary and Medium has same values as you see on the picture (figma). I capture the real greenMedium value from chrome debug pannel, so it should be fine.

File: src/assets/styles/vuetifyTheme.js (hunk 5 above)
Comment (hsurf22) [reply src/assets/styles/vuetifyTheme.js]: Screenshot from 2021-12-21 16-14-45
Screenshot from 2021-12-21 16-14-50

Same thing. I capture the real redMedium value from chrome debug pannel, so it should be fine.

**Comment by iluvboba** [comment]
moved to [#246 ](https://github.com/MyEtherWallet/mew-components/pull/246)

**Pull request closed without merging**